Samples/Sample.Net.ConsoleApp.App.Config/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Sample.Net.ConsoleApp.App.Config")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Sample.Net.ConsoleApp.App.Config")]
+[assembly: AssemblyCopyright("Copyright ©  2021")]
```

Comment:
Nit: extra space after the copyright.

Reply:
it is VS auto-generated file. let's leave it up to MS. plus, it has no significance for the demo app at all...